File: 217-contains-duplicate/contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        # nums.sort()
        # for i in nums:
        #    if(nums.count(i)>=2):
        #     return True
        # else:
        #     return False     
        nums.sort()
        count=1
        for i in range(len(nums)-1): 
            if(nums[i]==nums[i+1]):
                count+=1
            if(count>=2):
                return True
                break
        return False          
        

File: 217-contains-duplicate/contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Arrays.sort(nums);
        for(int i=0;i<nums.length;i++){
            if(i==nums.length-1){
                break;
            }
            if(nums[i]==nums[i+1]){
                return true;
            }
        }
        return false;
    }
}